homeassistant/components/goalzero/strings.json

```
@@ -22,5 +22,70 @@
       "invalid_host": "[%key:common::config_flow::error::invalid_host%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "backlight": {
+        "name": "Backlight"
+      },
+      "app_online": {
+        "name": "App online"
+      },
+      "input_detected": {
+        "name": "Input detected"
+      }
+    },
+    "sensor": {
+      "watts_in": {
+        "name": "Watts in"
+      },
+      "amps_in": {
+        "name": "Amps in"
+      },
+      "watts_out": {
+        "name": "Watts out"
+      },
+      "amps_out": {
+        "name": "Amps out"
+      },
+      "wh_out": {
+        "name": "Wh out"
+      },
+      "wh_stored": {
+        "name": "Wh stored"
+      },
+      "volts": {
+        "name": "Volts"
+      },
```

Shouldn't these be named by what they measure, not what unit they report?
I.e. something like:
```suggestion
      "watts_in": {
        "name": "Power in"
      },
      "amps_in": {
        "name": "Current in"
      },
      "watts_out": {
        "name": "Power out"
      },
      "amps_out": {
        "name": "Current out"
      },
      "wh_out": {
        "name": "Energy out"
      },
      "wh_stored": {
        "name": "Energy stored"
      },
      "volts": {
        "name": "Voltage"
      },
```